scopes/component/component-issues/multiple-envs.ts

```diff
@@ -3,7 +3,7 @@ import { ComponentIssue, ISSUE_FORMAT_SPACE } from './component-issue';
 export class MultipleEnvs extends ComponentIssue {
   description = 'multiple envs';
   solution =
-    'remove the old envs by setting them with "-" sign in the variants. see https://harmony-docs.bit.dev/aspects/variants#removing-aspects ';
+    'remove the old envs by setting them with "-" sign in the variants. see https://bit.dev/docs/envs/using-envs#component-must-have-a-single-env';
```

Comment:
Use base docs domain constant

Reply:
done (replaced all instances of hardcoded docs domain with the docs domain constant)